_scripts/tests/install.sh

```diff
@@ -22,6 +22,7 @@ if [ ! -z $CI ]; then
   echo "INJECTING VERSIONS!"
   echo "-------------------"
   pushd ../../ &> /dev/null
+  bumpver-if-set "deis-tests" "deis-e2e" "${WORKFLOW_E2E_SHA}"
```

Comment:
Yeah... Was thinking maybe removing the `_GIT_` on the others for brevity's sake via another PR was more what I would do.